rpcserver.go

```diff
@@ -813,9 +817,15 @@ func (r *rpcServer) ImportProof(ctx context.Context,
 
 	// Now that we know the proof file is at least present, we'll attempt
 	// to import it into the main archive.
-	err := r.cfg.ProofArchive.ImportProofs(ctx, &proof.AnnotatedProof{
-		Blob: in.ProofFile,
-	})
+	err := r.cfg.ProofArchive.ImportProofs(
+		ctx,
+		func(blockHeader wire.BlockHeader) error {
```

Comment:
Is there a reason to have this as nil right now vs. passing in the headerVerifier as done above?

Reply:
Fixed, thanks!